protocol/chainlib/chainlib.go

```diff
@@ -65,6 +65,7 @@ type ChainParser interface {
 	Activate()
 	UpdateBlockTime(newBlockTime time.Duration)
 	GetUniqueName() string
+	SetPolicyFromAddonAndExtensionSlice(policyInformation []string)
```

Comment:
let's not add another function, let's instead use a policy interface or an object that is partly filled